otx/algorithms/common/adapters/mmcv/utils.py

```diff
@@ -70,12 +70,15 @@ def config_from_string(config_string: str) -> Config:
 
 
 @check_input_parameters_type()
-def patch_data_pipeline(config: Config, base_dir: str):
+def patch_data_pipeline(config: Config, base_dir: str, base_data_pipeline: str = ""):
     """Replace data pipeline to data_pipeline.py if it exist."""
-    data_pipeline_path = os.path.join(base_dir, "data_pipelne.py")
+    data_pipeline_path = os.path.join(base_dir, "data_pipeline.py")
     if os.path.exists(data_pipeline_path):
         data_pipeline_cfg = Config.fromfile(data_pipeline_path)
         config.merge_from_dict(data_pipeline_cfg)
+    elif os.path.exists(base_data_pipeline):
+        data_pipeline_cfg = Config.fromfile(base_data_pipeline)
+        config.merge_from_dict(data_pipeline_cfg)
```

Comment:
how about using `data_pipeline.py` when `base_data_pipeline' doesn't exist?
I meant that we can use
if os.path.exists(base_data_pipeline):
 ...
elif os.path.exists(os.path.join(base_dir, "data_pipeline.py")):
 ...
else:
 raise error

Reply:
First of all, I prioritized `data_pipeline.py` in the template folder. If the file does not exist, it is configured to look for `base_data_pipeline`.

I think both have pros and cons, do you think `base_data_pipeline` should be given priority? 
+
There seems to be a way to combine both to find only `base_data_pipeline` without finding `data_pipeline.py` in template (This can be replaced with just `base_data_pipeline_path="./data_pipeline.py"`.) in the template folder.

Reply:
I modified things related to this based on `data_pipeline_path` (rename base_data* -> data_).